code/modules/mob/living/carbon/superior_animal/life.dm

```diff
@@ -1,5 +1,5 @@
 /mob/living/carbon/superior_animal/proc/check_AI_act()
-	if ((stat != CONSCIOUS) || !canmove || resting || lying || stasis || AI_inactive)
+	if ((stat != CONSCIOUS) || !canmove || resting || lying || stasis || AI_inactive || client)
 		stance = HOSTILE_STANCE_IDLE
 		target_mob = null
 		walk(src, 0)
```

Comment:
uhm

Reply:
??

Reply:
this is the only change to the proc life of the mobs that I have seen. But I already tested it and there is nothing wrong. So, I don't see that it is destroying the MC.